README.md

```diff
@@ -1149,6 +1149,20 @@ Extra IntelliJ plugins
 
         * Run that configuration.
 
+Linux inotify
+-------------
+
+We have encountered some issues with Linux's *inotify* when running any of the Workbench's in Super Dev Mode from within InteliJ.
+
+Error message `User limit of inotify instances reached or too many open files` has been observed with Ubuntu 15.04 (but not with Fedora 22). Should you encounter this issue you will need to reconfigure your *inotify* settings.
+
+Add the following to `/etc/sysctl.conf` and then run `sudo sysctl -p`:
+
+`fs.inotify.max_user_watches = 524288`
+
+`fs.inotify.max_user_instances = 524288`
+
+
```

Comment:
There's a related issue where the IntelliJ GWT tooling is opening too many files (as opposed to inotify instances). This can be fixed like this:

user soft nofile 4096
user hard nofile 10240